llvm/test/MC/X86/apx/long-instruction-err.s

```diff
@@ -0,0 +1,25 @@
+# RUN: not llvm-mc -triple x86_64 -show-encoding %s 2>&1 | FileCheck %s
+
+# CHECK: error: the size of instruction must be no longer than 15
```

Comment:
Test the file/line infomration like 
`[[#@LINE+2]]:34: error:`

Reply:
Good suggestion. Done